### PR TITLE
display distinct list of courses taught on /professor

### PR DESCRIPTION
**Pull request by nsandler1**

### CODE REVIEW

File: home/views/professor.py [hunk 1]
```diff
@@ -33,6 +33,7 @@ def get(self, request, slug):
             Course.recent
             .filter(professors__pk=professor.pk)
             .order_by("name")
+            .distinct()
```
Comment (tybug) [inline home/views/professor.py:36]: why is this required? surely this is a no-op since all courses are already distinct by pk, as we're not doing any joins.

Comment (nsandler1) [reply home/views/professor.py:36]: there was an issue where the courses taught in the review form and the list of courses taught in the top right corner of /professor generated a course for every entry in `ProfessorCourse` where `professor_id=professor.pk`. This was a problem for professors that taught classes across multiple semesters because such professors appear in `ProfessorCourse` once for each semester they taught a course.
